app/src/main/java/org/simple/clinic/drugs/selection/custom/CustomDrugEntryEffectHandler.kt

```diff
@@ -86,6 +87,19 @@ class CustomDrugEntryEffectHandler @AssistedInject constructor(
     }
   }
 
+  private fun removeDrugFromPrescription(): ObservableTransformer<RemoveDrugFromPrescription, CustomDrugEntryEvent>? {
+    return ObservableTransformer { effects ->
+      effects
+          .observeOn(schedulersProvider.io())
+          .map { it.drugUuid }
+          .flatMap {
```

Comment:
It won't make any difference here because `PrescriptionRepository#softDeletePrescription` returns a `Completable` which is then followed up with a cold `Observable` that only emits 1 item, but `flatMap` should be used only in very specific scenarios. We should be using `switchMap` by default unless we specifically need `flatMap`.

For what it's worth, we should be converting these methods to synchronous calls anyway, so the need for using the `Rx` operators will be removed.